extensions/formats/stanag4676/service/src/main/java/mil/nga/giat/geowave/types/stanag4676/service/rest/Stanag4676ImageryChipService.java

```diff
@@ -44,12 +45,25 @@
 import mil.nga.giat.geowave.format.stanag4676.image.ImageChipUtils;
 import mil.nga.giat.geowave.service.ServiceUtils;
 
+import org.apache.commons.io.IOUtils;
 import org.apache.log4j.Logger;
+import org.jcodec.codecs.vpx.NopRateControl;
+import org.jcodec.codecs.vpx.RateControl;
+import org.jcodec.codecs.vpx.VP8Encoder;
+import org.jcodec.common.FileChannelWrapper;
+import org.jcodec.common.NIOUtils;
+import org.jcodec.common.model.ColorSpace;
+import org.jcodec.common.model.Picture;
+import org.jcodec.common.model.Size;
+import org.jcodec.containers.mkv.muxer.MKVMuxer;
+import org.jcodec.containers.mkv.muxer.MKVMuxerTrack;
+import org.jcodec.scale.AWTUtil;
+import org.jcodec.scale.RgbToYuv420p;
 
 import com.google.common.io.Files;
-import com.xuggle.mediatool.IMediaWriter;
-import com.xuggle.mediatool.ToolFactory;
-import com.xuggle.xuggler.ICodec;
+//import com.xuggle.mediatool.IMediaWriter;
+//import com.xuggle.mediatool.ToolFactory;
```

Comment:
feel free to delete the various commented lines of code - we have revision history if we wanted to revisit xuggler